src/ContactPeople/ContactPeopleView.js

```diff
@@ -185,13 +208,18 @@ class ContactPeopleView extends React.Component {
     );
   }
 
+  getContacts() {
+    const { parentResources } = this.props;
+    const data = ((parentResources || {}).contacts || {}).records || [];
```

Comment:
IMO, this is more common approach:
```suggestion
    const data = get(parentResources, 'contacts.records') || [];
```